guides/server/telemetry.md

```diff
@@ -191,6 +191,16 @@ LiveView currently exposes the following [`telemetry`](https://hexdocs.pm/teleme
             changed?: boolean
           }
 
+  * `[:phoenix, :live_view, :component_destroyed]` - Dispatched by a `Phoenix.LiveView`
```

Comment:
Below we name it `:phoenix, :live_component`. Perhaps we should do the same here:

```suggestion
  * `[:phoenix, :live_component, :destroyed]` - Dispatched by a `Phoenix.LiveView`
```

Also, can you include the `:component` name as well? Or we don't have it available?

Reply:
The difference here is that it actually is a LiveView related event with the metadata containing the LiveView's socket, not the LiveComponent one. We could change it, but if you then want to find the LiveView that this is actually related to, you'd need to do it based on `self()` - or maybe I'm missing something.

Reply:
we can get id
https://github.com/phoenixframework/phoenix_live_view/blob/92bd1e6990ed75dc3f845497b06d5387a42d7b63/lib/phoenix_live_view/channel.ex#L1534-L1536

Reply:
I see. My preference them would be to make it like all other components events, and we can attach the socket as additional metadata (which would be used by LiveDebugger).

Reply:
@josevalim updated: https://github.com/phoenixframework/phoenix_live_view/pull/3777/commits/5858875b8a853e440e429b62df505fdb0316544c